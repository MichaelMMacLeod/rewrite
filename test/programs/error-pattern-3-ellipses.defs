(def ($x .. $y .. $z ..) constructor)