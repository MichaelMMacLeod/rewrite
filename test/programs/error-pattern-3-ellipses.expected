error[E006]: too many ellipses in single term of pattern
./test/programs/error-pattern-3-ellipses.defs:1:6
   |
 1 | (def ($x .. $y .. $z ..) constructor)
   |      ^^^^^^^^^^^^^^^^^^^ this term has more than one ellipsis
./test/programs/error-pattern-3-ellipses.defs:1:16
   |
 1 | (def ($x .. $y .. $z ..) constructor)
   |                ^^ ellipsis #2
./test/programs/error-pattern-3-ellipses.defs:1:22
   |
 1 | (def ($x .. $y .. $z ..) constructor)
   |                      ^^ ellipsis #3
help: each term in a definitions's pattern term may have at most one ellipsis
